xml/common_intro_available_doc_i.xml

```diff
@@ -20,134 +20,49 @@
-   <link os="sles;sled" xlink:href="https://documentation.suse.com/"/><link os="osuse" xlink:href="http://doc.opensuse.org/"/>,
-   where you can also find the latest updates, and browse or download the documentation in various formats.
-   The latest documentation updates are usually available in the English version of the documentation.
-  </para>
- </note>
- <note>
-  <title>Manual Pages</title>
-  <para>
-   Many commands are described in detail in their <emphasis>manual
-   pages</emphasis>. You can view manual pages by running the
-   <command>man</command> command followed by a specific command name. If the
-   <command>man</command> command is not installed on your system, install it
-   by running <command>zypper install man</command>.
-  </para>
- </note>
- <para>
-  The following documentation is available for this product:
- </para>
-
- <variablelist>
-  <varlistentry os="sles;sled">
-   <term><xref linkend="art-sle-installquick"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_installquick;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;sled">
-   <term><xref linkend="book-sle-deployment"/>
-   </term>
-   <listitem>
-    <para>
-     <remark>taroth 2016-05-20: check with cwickert if abstract is
-     still correct after the DG rewrite </remark>
-     &abstract_deployment;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="osuse">
-   <term><xref linkend="book-opensuse-startup"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_osuse_startup;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry>
-   <term><xref linkend="book-sle-admin" os="sles;sled"/><xref linkend="book-opensuse-reference" os="osuse"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_admin;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;osuse">
-   <term><xref linkend="book-virt"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_virtualization;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles">
-   <term><xref linkend="book-storage"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_storage;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry os="sles;osuse">
-   <term><xref linkend="book-autoyast"/>
-   </term>
-   <listitem>
-    <para>
-     &abstract_autoyast;
-    </para>
-   </listitem>
-  </varlistentry>
-  <varlistentry>
-   <term><xref linkend="book-security"/>
-   </term>
-   <listitem>
+<variablelist>
+ <varlistentry>
+  <term>Online documentation</term>
+  <listitem>
+   <para>
+    The online documentation for this product is available at
+<!--taroth 2021-03-18: keep the profiled links in one line,
+    moving them to separate lines would create a whitespace before
+    the full stop-->
+    <link os="sles" xlink:href="https://documentation.suse.com/#sles"/><link os="sled" xlink:href="https://documentation.suse.com/#sled"/><link os="osuse" xlink:href="http://doc.opensuse.org/"/>.
```

Comment:
Might make sense to use an entity for the URL here -- which could be defined along with &productname; and &productnumber; -- this would allow adding the product version number without too much danger of it later being forgotten. In the release notes, we already use `:doc-url: https://documentation.suse.com/sles/15-SP3` -- so, for consistency, &doc-url; would make most sense to me.

Reply:
Thanks, I see your point, but as none of the other URLs contains entities for several reasons (outlined here https://github.com/SUSE/doc-sle/pull/579#issue-402627684) I would not want to introduce them now.

Reply:
Ugh. I had just about forgotten about this kerfuffle. :/ In any case, this would be the entire URL in an entity, not just part of the URL.

The way it is right now, when you click this link, you still need to select a product version, which I think is a click we should ideally save those users, because we know what product version they are looking at currently.